docs/ru/query_language/functions/string_functions.md

```diff
@@ -42,7 +42,30 @@
 Возвращает 1, если набор байт является корректным в кодировке UTF-8, 0 иначе.
 
 ## toValidUTF8
-Возвращает строку, где все некорректные UTF-8 символы заменены на заменяющий символ `�` (U+FFFD). Если много некорректных символов идут подряд, схлопывается до одного заменяющего символа.
+
+Заменяет некорректные символы UTF-8 на символ `�` (U+FFFD). Все идущие подряд некорректные символы схлопываются в один заменяющий символ.
+
+```
+toValidUTF8( input_string )
+```
+
+Параметры:
+
+- input_string — произвольный набор байтов, представленный как объект типа [String](../../data_types/string.md).
+
+Возвращаемое значение: Корректная строка UTF-8.
+
+### Пример
+
+```sql
+SELECT toValidUTF8('\x61\xF0\x80\x80\x80')
+```
+```text
+┌─toValidUTF8('a����')─┐
+│ a�                   │
```

Comment:
a�b

Reply:
Yes https://github.com/yandex/ClickHouse/pull/5348/commits/fb7f6550157c937a98d62420b0bcccbc64e03cef

Reply:
It doesn't seem you've fixed it.